bundles/org.eclipse.mita.program/src/org/eclipse/mita/program/generator/ProgramDslGenerator.xtend

```diff
@@ -134,18 +133,24 @@ class ProgramDslGenerator extends AbstractGenerator implements IGeneratorOnResou
 	}
 	
 	override doGenerate(ResourceSet input, IFileSystemAccess2 fsa) {
+		doGenerate(input, fsa, [ it.URI?.segment(0) == 'resource' ]);
+	}
+	
+	override doGenerate(ResourceSet input, IFileSystemAccess2 fsa, Function1<Resource, Boolean> includeInBuildPredicate) {
```

Comment:
In Xtend, `Function1<A, B>` can be written as `(A) => B`

Reply:
Good point. Unfortunately the interface we're implementing with `doGenerate` is expressed in Java and to be able to implement it, we need to using `Function1`.

Reply:
I see. Strange of Xtext to duplicate functionality from Java...